### PR TITLE
feat: powered by lenders in borrower profile

### PR DESCRIPTION
**Pull request by christian14b**
image

### CODE REVIEW

File: test/unit/specs/components/BorrowerProfile/PoweredByLenders.spec.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import { render } from '@testing-library/vue';
+import PoweredByLenders from '@/components/BorrowerProfile/PoweredByLenders';
+
+const participants = {
```
Comment (dyersituations) [inline test/unit/specs/components/BorrowerProfile/PoweredByLenders.spec.js:4]: Could this also use the new mock data?

File: src/components/BorrowerProfile/PoweredByLenders.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<template>
+	<div class="tw-flex tw-items-center tw-mb-1.5">
+		<div class="tw-flex tw-shrink-0">
+			<img
```
Comment (dyersituations) [inline src/components/BorrowerProfile/PoweredByLenders.vue:4]: Can we use `KvUserAvatar`?

File: test/unit/specs/components/BorrowerProfile/PoweredByLenders.spec.js (hunk 1 above)
Comment (roger-in-kiva) [reply test/unit/specs/components/BorrowerProfile/PoweredByLenders.spec.js:4]: We already have contributing lender on the page, why using mock data?

Comment (dyersituations) [reply test/unit/specs/components/BorrowerProfile/PoweredByLenders.spec.js:4]: I was referring to the storybook mock data in this changeset, though I guess we don't typically share mocks between storybook and jest, so disregard.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**